docs/using.rst

```diff
@@ -516,6 +516,13 @@ can run on a regular basis, like every week or every day). In that case,
 you are likely to want to use the ``-q`` or ``--quiet`` quiet flag to
 silence all output except errors.
 
+.. seealso:: Many of the certbot clients obtained through a
+   distribution come with automatic renewal out of the box,
+   such as Debian and Ubuntu versions installed through `apt`,
+   CentOS/RHEL 7 through EPEL, etc.  See `Automated Renewals`_
+   for more details, or consult your distribution's
```

Comment:
Is it accurate to say that most distribution-packaged Certbots have documentation somewhere about their cron renewals? I don't see any in the Ubuntu package. I think if the current state is "mostly undocumented" we should leave out this mention of the distribution's documentation.

Reply:
The Ubuntu package is the Debian package, so they ship both cron and systemd.  Apparently CentOS and RHEL both do as well, and the combination of those distributions and their derivatives is the bulk of Linux installations out there.

Reply:
Right, but do the Debian/Ubuntu packages ship documentation? This PR is saying "check the OS documentation," but I think it doesn't exist.

Reply:
I think this is still unresolved. Could you just remove the part that says "or consult your distribution's documentation"?